api/rest/mantisbt_openapi.yaml

```diff
@@ -0,0 +1,1733 @@
+openapi: 3.0.0
+info:
+  title: Mantis Bug Tracker REST API
+  version: 2.27.0
```

Comment:
Suggest adding a *description* key, see example 
https://github.com/dregad/mantisbt/blob/7199f739cf7cf9b116a918522d1c03f45efe25e3/api/rest/openapi.yaml#L9-L20

The above was originally generated by Postman export IIRC. This is Markdown text.

Also, a *contact* key would be nice
```yaml
  contact:
    name: MantisBT Team
    email: mantisbt-dev@lists.sourceforge.net
    url: https://mantisbt.org
```